src/modules/layout/layout.component.html

```diff
@@ -41,4 +41,16 @@
   </stache-layout-sidebar>
 </template>
 
+<template #actionButtonLayout>
+  <stache-layout-action-button
+    pageTitle="{{pageTitle}}"
+    [showBackToTop]="showBackToTop"
+    [showBreadcrumbs]="showBreadcrumbs"
+    [breadcrumbsRoutes]="breadcrumbsRoutes"
+    [actionButtonRoutes]="actionButtonRoutes">
```

Comment:
Agreed. The amount of routes we are passing in is getting crazy. We should come up with a more maintainable pattern.